src/main/java/cz/cvut/kbss/study/dto/PatientRecordDto.java

```diff
@@ -22,8 +22,8 @@ public class PatientRecordDto extends AbstractEntity implements HasOwlKey {
     private String localName;
 
     @ParticipationConstraints(nonEmpty = true)
-    @OWLObjectProperty(iri = Vocabulary.s_p_has_author)
-    private URI author;
+    @OWLObjectProperty(iri = Vocabulary.s_p_has_author, fetch = FetchType.EAGER)
+    private User author;
```

Comment:
FYI, I would consider UserDto instead of User. Do not need to resolve it, let's consider it next  time ;)